api/preprint_providers/serializers.py

```diff
@@ -108,6 +109,9 @@ class PreprintProviderSerializer(JSONAPISerializer):
     class Meta:
         type_ = 'preprint_providers'
 
+    def get_highlighted_subjects_exists(self, obj):
```

Comment:
Minor, and should have thought of this yesterday: change this to `get_has_highlighted_subjects()` so that it's consistently named.